webpack.config.js

```diff
@@ -33,7 +33,10 @@ if (__DEV__) {
 }
 
 const plugins = [
-  new webpack.DefinePlugin({ __DEV__ }),
+  new webpack.DefinePlugin({
+    __DEV__,
+    'process.env.npm_package_version': JSON.stringify(process.env.npm_package_version)
```

Comment:
```suggestion
    __VERSION__: JSON.stringify(process.env.npm_package_version),
```
you'll also have add this to the globals in the eslintrc https://github.com/Flexget/webui/blob/develop/.eslintrc.js#L30